tools/styles/ce/stylesheet.css

```diff
@@ -0,0 +1,64 @@
+/* Javadoc style sheet */
```

Comment:
To make naming consistent, this should be named after the reference doc it styles (i.e., javadoc.css).